wiki/Hit_Objects/en.md

```diff
@@ -27,6 +30,8 @@ Sliders are often mapped to long sounds in the music. They are also used to carr
 Spinner
 -------
 
-<img src="Spinner.jpg" title="fig:A spinner in the default skin." alt="A spinner in the default skin." width="200" /> A Spinner is a circle that takes up the entire height of the play area and has a [Spinner Metre](Glossary#Spinner_Metre "wikilink") on either side. A large circular outline similar to a [Approach Circle](Glossary#Approach_Circle "wikilink") appears on the Spinner’s border and gradually converges on its centre. The player should hold the mouse button down and spin in either a clockwise or anti-clockwise direction. As long as the mouse button is held down, the cursor is locked on the Spinner’s area, so the player need not worry about going outside it. The Spinner is cleared if all the levels of the Spinner Metre are filled before its Approach Circle reaches the centre. Depending on the difficulty settings of each [beatmap](Glossary#Beatmap "wikilink"), a different number of spins are required to fill each level of the Spinner Metre (worth 100 points). If the player fills the Spinner Metre before the Approach Circle reaches the Spinner’s centre, each additional spin before time runs out will award the player with 1,000 bonus points (each receiving a bonus multiplier depending on the player’s current [Combo](Combo "wikilink")). Spinning a Spinner provides a continuous minor boost to the [Life Bar](Glossary#Life_Bar "wikilink") depending on the spinning speed.
+![A spinner in the default skin.](Spinner.jpg "A spinner in the default skin.")
+
+A Spinner is a circle that takes up the entire height of the play area and has a [Spinner Metre](/wiki/Glossary) on either side. A large circular outline similar to a [Approach Circle](/wiki/Glossary) appears on the Spinner’s border and gradually converges on its centre. The player should hold the mouse button down and spin in either a clockwise or anti-clockwise direction. As long as the mouse button is held down, the cursor is locked on the Spinner’s area, so the player need not worry about going outside it. The Spinner is cleared if all the levels of the Spinner Metre are filled before its Approach Circle reaches the centre. Depending on the difficulty settings of each [beatmap](/wiki/Glossary), a different number of spins are required to fill each level of the Spinner Metre (worth 100 points). If the player fills the Spinner Metre before the Approach Circle reaches the Spinner’s centre, each additional spin before time runs out will award the player with 1,000 bonus points (each receiving a bonus multiplier depending on the player’s current [Combo](/wiki/Glossary)). Spinning a Spinner provides a continuous minor boost to the [Life Bar](/wiki/Glossary) depending on the spinning speed.
```

Comment:
this talks about v1 skin behavior (and Spinner Metre isn't defined in the glossary; if need be, I could add it to my local)
...
I just read the line below, but really, this line should be updated instead